webapp/channels/src/components/channel_notifications_modal/channel_notifications_modal.tsx

```diff
@@ -461,14 +461,14 @@ export default class ChannelNotificationsModal extends React.PureComponent<Props
                                         globalNotificationLevel={currentUser.notify_props ? currentUser.notify_props.desktop : NotificationLevels.ALL}
                                         globalNotificationSound={(currentUser.notify_props && currentUser.notify_props.desktop_notification_sound) ? currentUser.notify_props.desktop_notification_sound : 'Bing'}
                                         isNotificationsSettingSameAsGlobal={isNotificationsSettingSameAsGlobal}
-                                        onChange={this.handleUpdateDesktopNotifyLevel}
-                                        onChangeThreads={this.handleUpdateDesktopThreadsNotifyLevel}
-                                        onChangeDesktopSound={this.handleUpdateDesktopSound}
-                                        onChangeNotificationSound={this.handleUpdateDesktopNotifySound}
+                                        onChange={this.handleUpdateDesktopNotifyLevel as (value?: string) => void}
+                                        onChangeThreads={this.handleUpdateDesktopThreadsNotifyLevel as (value?: string) => void}
+                                        onChangeDesktopSound={this.handleUpdateDesktopSound as (value?: string) => void}
+                                        onChangeNotificationSound={this.handleUpdateDesktopNotifySound as (value?: string) => void}
```

Comment:
i understand why this is needed but I suggest another approach to not have the explicit conversion.
`onChange: (value: string | any) .......`

Reply:
Sure, thanks for suggestion!